packages/ollydbg.vm/ollydbg.vm.nuspec

```diff
@@ -3,7 +3,6 @@
   <metadata>
     <id>ollydbg.vm</id>
```

Comment:
Does an update of only the `.nuspec` file require a version update? If so, I need to modify the linter.

Reply:
Yes. Imagine we update the authors because they are wrong. Then the updated package won't be pushed if the version is not updated.

The previous CI check is a quick fix and is not covering this case.

Reply:
updated in 26653af